deepspeed/runtime/zero/stage3.py

```diff
@@ -73,16 +65,34 @@ def lcm(x, y):
     return x * y // gcd(x, y)
 
 
+def debug_rank0(message: str) -> None:
+    if dist.get_rank() == 0:
+        logger.debug(message)
+
+
+def get_cuda_mem_allocated_str() -> str:

```

Comment:
This looks like a subset of [see_memory_usage](https://github.com/microsoft/DeepSpeed/blob/c3f1d82b07b7541ee03961b07959b9f38894de5e/deepspeed/runtime/utils.py#L812). Is it needed?

Reply:
i think we can get rid of it

Reply:
removed